activiti-cloud-messages-service/integration-tests/src/test/java/org/activiti/cloud/messages/integration/tests/rb/MultipleRbMessagesIT.java

```diff
@@ -127,6 +116,10 @@ public ReceiveMessageCmdExecutor receiveMessageCmdExecutor(ProcessAdminRuntime p
 
     @BeforeAll
     public static void setUp() {
+        KeycloakContainerApplicationInitializer keycloakContainerApplicationInitializer = new KeycloakContainerApplicationInitializer();
```

Comment:
It is not needed for now but as a suggestion to lower the cognitive load we can use the `var` keyword in these cases, like 
```
var keycloakContainerApplicationInitializer = new KeycloakContainerApplicationInitializer();
```